meta-cli/src/typegraph/loader/mod.rs

```diff
@@ -123,7 +205,38 @@ impl Loader {
             Ok(String::from_utf8(p.stdout)?)
         } else {
             let stderr = String::from_utf8(p.stderr)?;
-            bail!("Python error:\n{}", stderr.red())
+            anyhow::bail!("Python error:\n{}", stderr.red())
+        }
+    }
+
+    #[cfg(not(feature = "typegraph-next"))]
+    pub async fn load_deno_module(&self, _path: &Path) -> Result<String> {
+        anyhow::bail!("deno modules not supported yet")
+    }
+
+    #[cfg(feature = "typegraph-next")]
+    fn get_load_command(module_type: ModuleType, path: &Path) -> Command {
+        let vars: HashMap<_, _> = env::vars().collect();
+        match module_type {
+            ModuleType::Python => {
+                let mut command = Command::new("python3");
+                command
+                    .arg(path.to_str().unwrap())
+                    .envs(vars)
+                    .env("PYTHONUNBUFFERED", "1")
+                    .env("PYTHONDONTWRITEBYTECODE", "1");
+                command
+            }
+            ModuleType::Deno => {
+                let mut command = Command::new("deno");
```

Comment:
The typescript SDK will be also published on NPM, so we may need to have a fallback on node as loader (should be detectable based on the import format)

Reply:
That would be JavaScript then instead of TypeScript? Or shall we transpile the code on our own?

Reply:
I meant, the typegraph definition module.